src/pages/LandingPage.js

```diff
@@ -86,6 +90,26 @@ const LandingPage = () => {
       }, ['See this article']),
       ' for a summary of available resources.'
     ]),
+    isDataBrowserVisible() && div({
+      style: {
+        backgroundColor: colors.primary(),
+        backgroundImage: `url(${terraHero})`, backgroundSize: 'cover', borderRadius: 5,
+        display: 'flex', flexDirection: 'row', justifyContent: 'space-between', alignItems: 'center',
+        boxShadow: '0 2px 5px 0 rgba(0,0,0,0.35), 0 3px 2px 0 rgba(0,0,0,0.12)',
+        color: 'white', padding: '2rem 1rem',
+        width: 'calc(675px + 2rem)', // 3 card widths + 2 card margins to line up with the content directly above
+        marginTop: 15
+      }
+    }, [
+      div([
+        h2({ style: { fontSize: 18, fontWeight: 500, lineHeight: '22px', margin: 0 } }, ['BETA Data Catalog']),
+        'Preview the Data Catalog and provide valuable feedback'
```

Comment:
Ending this sentence with a period would make it consistent with the COVID section.